<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerminaldesktopList</class>
 <widget class="QDialog" name="TerminaldesktopList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>476</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>312</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
border:1px solid #e9e9e9;
border-radius:4px;
}
QDialog#DesktopListDialog{
	background-image:url(&quot;:image/resource/image/skin_expand.png&quot;);
}

QWidget#scrollAreaWidgetContents{
	padding:0px;
	margin-left:0px;
	margin-right:2px;
	border-left:1px solid #cccccc;
	border-right:1px solid #cccccc;
	padding-top: 2px;
	padding-bottom: 2px;
	background: #fafafa;
}

QScrollBar:vertical {
border: 0px groove #fafafa;
/*padding-right:2px;*/
/*padding-left:1px;*/
/*margin-right:1px;*/
/*background: #fafafa;/*#474747*/
/*background-image:url(&quot;:image/resource/image/skin_expand.png&quot;);*/
/*width: 16px;*/
background: #ffffff;
width: 14px;
}
QScrollBar::handle:vertical {
background: #b5b5b5;
border:2px solid #b5b5b5;
/*border-color:#cccccc*/
border-radius:4px;
padding:4px;
padding-right:3px;
margin-left:3px;
margin-right:3px;
/*background-image:url(&quot;:image/resource/image/skin_expand.png&quot;);*/
/*min-height: 20px;*/
width: 8px;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
background:#fafafa;
/*background: #474747;*/
/*background-image:url(&quot;:image/resource/image/skin_expand.png&quot;);*/
}

QScrollBar::add-line:vertical {
background:#fafafa;
}

QScrollBar::sub-line:vertical {
background:#fafafa;
}

/*QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
border: 0px solid grey;
width: 3px;
height: 3px;
background: black;*/
/*background-image:url(&quot;:image/resource/image/skin_expand.png&quot;);*/
/*}*/



</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_top">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_corner">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_title">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="scrollWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>13</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QScrollArea" name="scrollArea_desktopList">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">/*QScrollArea{
	border-top: 1px solid #cccccc;
	border-bottom: 1px solid #cccccc;
}*/</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>292</width>
           <height>408</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_bottom">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>16</number>
     </property>
     <property name="rightMargin">
      <number>16</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkall">
       <property name="text">
        <string>CheckAll</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushBtn_poweroff">
       <property name="text">
        <string>PowerOff</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushBtn_restart">
       <property name="text">
        <string>Restart</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushBtn_msg">
       <property name="text">
        <string>Message</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
